x-pack/plugins/infra/public/utils/get_chart_theme.ts

```diff
@@ -1,40 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { Theme, PartialTheme, LIGHT_THEME, DARK_THEME, SettingsProps } from '@elastic/charts';
-
-// TODO use the EUI charts theme see src/plugins/charts/public/services/theme/README.md
-export function getChartTheme(isDarkMode: boolean): Theme {
-  return isDarkMode ? DARK_THEME : LIGHT_THEME;
-}
-
-const TIMELINE_LIGHT_THEME: PartialTheme = {
-  crosshair: {
-    band: {
-      fill: '#D3DAE6',
```

Comment:
This is to color the vertical line that appears when hovering on the chart. The theme's default color is darker than this.

<img width="1143" alt="image" src="https://github.com/elastic/kibana/assets/2767137/6b975d44-ff8b-48ba-a5e3-0dade2a81423">